t04_refactored.py

```diff
@@ -211,9 +211,41 @@ def team_13_adv(username, delay):
 
 
 def team_14_adv(username, delay):
-    pass
-    # TODO Add your code here
-    return True
+    '''https://docs.google.com/document/d/1scXuIBzrGZxe0pZ-vEulOikC7Vc0QR5suwVhLVz6M9o/edit
+        Chris Georgiev
+        Dylan Roland
+        Kamau Clark'''
+
+    isDead = False
+
+    direction = input("Hey " + username + ", do you want to go North(1) or South(2)?: ")
+
+    if direction == "1":
+        print("You didn't die. In fact, you made it out of the cave and back into society. Yay.")
+        sleep(delay)
+
+    elif direction == "2":
+        print("You unfortunately fell off a cliff and are about to die.")
+        isDead = True
+        sleep(delay)
+
+    else:
+        print("You're still in the cave, and no closer to being out of it. Sucks to be you.")
+        sleep(delay)
+
+    if isDead == True:
+        numbertest = input("Pick a number 1-10 to try and survive.")
+        numbertest = float(numbertest)
+
+        if numbertest > 5 and numbertest <= 10:
+            isDead == False
+            print("You got lucky this time.")
+
+        else:
+            print("Unlucky, you're dead. Very sad")
+            quit()
```

Comment:
Here, `return False` to indicate they are now dead.

Reply:
Nevermind... instead, DONT `quit()` (or other functions won't get a chance to run).